app/Models/Entry.php

```diff
@@ -87,9 +87,11 @@ public function link() {
 	public function date($raw = false) {
 		if ($raw) {
 			return $this->date;
-		} else {
-			return timestamptodate($this->date);
 		}
+		return timestamptodate($this->date);
+	}
+	public function machineReadableDate() {
+		return @date ('Y-m-d\TH:i:s', $this->date);
```

Comment:
Wouldn't the standard `date('c', ...)` work, or even `gmdate('c', ...)`?

Reply:
It works and it's easier to understand. But I found an even clearer way :)